From 3890e27d6684f03b53f2910e0f3e48d0fc76d19c Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:05 -0400
Subject: [redhat] kabi: Adding symbol mempool_create (mm/mempool.c)

Message-id: <07c9e59552f532947972c543792bccbdd27f60fb.1410510773.git.poros@redhat.com>
Patchwork-id: 96095
O-Subject: [PATCH RHEL7.1 125/233] kabi: Adding symbol mempool_create (mm/mempool.c)
Bugzilla: 1062361 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098,1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Used to create memory pool for internal data structures
    for IOs as we cannot rely on other types of memory in IO context.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/mempool_create
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/mempool_create
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/mempool_create

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/mempool_create b/redhat/kabi/kabi-module/kabi_ppc64/mempool_create
new file mode 100644
index 0000000..ef9c74a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/mempool_create
@@ -0,0 +1,2 @@
+#1-
+0x8de0b5ac mempool_create vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/mempool_create b/redhat/kabi/kabi-module/kabi_s390x/mempool_create
new file mode 100644
index 0000000..a931df0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/mempool_create
@@ -0,0 +1,2 @@
+#1-
+0x4557907d mempool_create vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/mempool_create b/redhat/kabi/kabi-module/kabi_x86_64/mempool_create
new file mode 100644
index 0000000..f787040
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/mempool_create
@@ -0,0 +1,2 @@
+#1-
+0x3f71faa1 mempool_create vmlinux EXPORT_SYMBOL
-- 
1.7.1